www/demo/index.html

```diff
@@ -15,7 +15,8 @@
       <a href="https://github.com/ikr/react-hotels-on-map">GitHub</a>
     </p>
 
-    <script src="https://maps.googleapis.com/maps/api/js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCIvWfrF-Z_nVANRql9gfH848PYcjoVLho"
```

Comment:
Why is that change necessary? Doesn't it work without the key? Whose key is that?